python/nnvm/top/symbol_array.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=invalid-name
+"""Symbol array object used by weight pre-pack"""
+import tvm
+
+_sym_arr_get = tvm.get_global_func("nnvm.compiler._symbol_array_get")
+_sym_arr_size = tvm.get_global_func("nnvm.compiler._symbol_array_size")
+
+class SymbolArray(object):
```

Comment:
We can just use symbol for this, because we can group multiple nodes into a single Symbol